Utilities/Mergers/test/test_ObjectStore.cxx

```diff
@@ -9,6 +9,7 @@
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
 
+#include <TH1.h>
```

Comment:
why is it needed here? I would rather add `<TH1F.h>` next to the other ROOT histogram include, since you used that one in the new test case.